content/using/install.md

```diff
@@ -193,151 +195,17 @@ As mentioned previously, there are a few ways to acquire a planet. All methods,
-
-Now you're ready to explore the network and make your ship easily accessible from a remote browser. Jump to the [System Administration](#admin) section to do so.
-
-### Option B: Local Installation {#local}
-
-You can run an Urbit on your laptop, your desktop, or an old tower in your closet. There are fewer steps than installing it on a cloud service, but it won't be as slick of an experience in the long run.
-
-The Urbit binary runs on a Unix-like operating system. We have different installation instructions for different platforms. To install and run Urbit OS, the Urbit OS and its binary, run the commands that are listed for your operating system.
-
-Once you've followed the appropriate install instructions, you can check if everything went right by running the `./urbit` command. Installation was successful if you get a block of output that begins with the line below:
-
-```
-Urbit: a personal server operating function
-```
-
-#### macOS
-
-```sh
-curl -O https://bootstrap.urbit.org/urbit-v0.10.8-darwin.tgz
-tar xzf urbit-v0.10.8-darwin.tgz
-cd urbit-v0.10.8-darwin
-./urbit
-```
-
-#### Linux (64-bit)
-
-```sh
-curl -O https://bootstrap.urbit.org/urbit-v0.10.8-linux64.tgz
-tar xzf urbit-v0.10.8-linux64.tgz
-cd urbit-v0.10.8-linux64
-./urbit
-```
-
-To access your Urbit via HTTP on port 80 on Ubuntu, you may need to run the following:
-
-```sh
-sudo apt-get install libcap2-bin
-sudo setcap 'cap_net_bind_service=+ep' /path/to/urbit
-```
-(Where `urbit` is the urbit executable downloaded with `curl` prior)
-
-#### Windows
+## Hosting
 
-Please note that this method of installing Urbit is more experimental, and we may not be able to assist you if you encounter issues related to WSL 2, but these instructions have been tested and verified for WSL 2 + Ubuntu 18.04 LTS.
+There are two supported ways of running an Urbit ship: using a cloud service, or running it locally on your own machine. Most users run things locally at first, but we recommend using a cloud service for Urbit because it allows your ship to be accessed from anywhere on multiple devices. Hosting your ship in the cloud also allows it to always be online and ready for OTA updates. It's technically possible to run your Urbit ship on a home server, but ISPs often restrict this to business plans and opening up your home network to the internet can be unsafe if done improperly.
 
-Urbit cannot run on Windows itself, but there is a convenient way to run a Linux distro using the [Windows Subsystem for Linux 2](https://en.wikipedia.org/wiki/Windows_Subsystem_for_Linux), an initiative by Microsoft to bring Linux to Windows 10. For Urbit to work, it is necessary to [install WSL 2](https://docs.microsoft.com/en-us/windows/wsl/wsl2-install) and not just WSL, since version 2 uses virtualization and the real Linux kernel under the hood.
+Using a cloud service costs money, but it's more convenient to use than a local install.
```

Comment:
Is a cloud service more convenient? It's more of a PITA in the short-term, but has all of the benefits listed above. I'm not sure this line is necessary.

Reply:
👍 I was tempted to delete it, but I think it was already there in the mass of docs I removed. Will delete.